### ISSUE TITLE
[Request] Add jquery.unobtrusive-ajax

### ISSUE THREAD

**Issue by jmhongithub** [cdnjs/cdnjs#12790]
**Library name:** jquery.unobtrusive-ajax
**Library description:** 
**Git repository url:** https://github.com/aspnet/jquery-ajax-unobtrusive
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** Apache License, Version 2.0 
**Official homepage:** https://github.com/aspnet/home
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer(please remove this section after you read it):
1. Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.
2. For the new library request issue, please make sure it's not a personal project, we have a basic requirement for the popularity, like 200 stars on GitHub or 800 downloads/month on npm registry.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

**Comment by sashberd** [comment]
Hi @jmhongithub 

The library doesn't have enough popularity(200 stars/watchers on GitHub and 800 downloads/month on npm registry are good examples). You can take a look at our issue rule for more details 😄

This issue will be closed if there is no any response in next 7 days, if there is any other possibility to add this lib, we may discuss to add it.

Thank you for your request!

**Comment by GerkinDev** [comment]
Related to #11976. Both deserve to be on CDNJS IMHO, as I said on the other thread. For posterity:

> The npm package has about 1700 downloads weekly, so it meet the requirements for inclusion in CDNJS. Moreover, even if it has less downloads/stars, most people use the local copy of this script, used on many ASP.NET sites. Offering the ability to use CDN would help many people using currently the local copy.

AFAIK, both asp.net Core & asp.net use the same version of this package.

**Comment by iMusashi** [comment]
The latest update by Visual Studio provides cdnjs as a standard reference for downloading files via cdn. It will be great, if the above library is added to the list. Currently, there is no other direct alternative to downloading files to a specific project folder except the latest tool Library Manager in Visual Studio. Thanks

**Comment by javiercn** [comment]
Hi,

I work on the ASP.NET team at Microsoft. I've created a pull request to add this library to cdnjs. I've included all the stats in the PR. https://github.com/cdnjs/cdnjs/pull/13029

It would be great if any project maintainer could help us get the library on to cdnjs or tell us what the next steps for this are. (In case there's anything wrong or missing on the PR)

There's also a duplicate issue for this in https://github.com/cdnjs/cdnjs/issues/11976

Thanks!
